setup.py

```diff
@@ -16,6 +16,7 @@
     'pandas>=1.1,<1.1.5',
     'scikit-learn>=0.23,<1',
     'scipy>=1.4.1,<2',
+    'numba>=0.50,<0.54',
```

Comment:
Why is this needed?

Reply:
`ERROR: numba 0.54.1 has requirement numpy<1.21,>=1.17, but you'll have numpy 1.21.3 which is incompatible.`
Which comes from `'sktime>=0.4,<0.6',` that do not cap `numba` https://github.com/alan-turing-institute/sktime/blob/main/setup.py which ends up pulling the latest release, that actually produces that error.

Reply:
Even with the new changes this still fails.

Reply:
I'm a bit confused about this @pvk-developer: So `numba==0.54.1` requires `numpy<1.21`, but `numba<0.54` works well with `numpy==1.21.3`?

Reply:
Yes, looks like numba `0.54` is the only version that does not accept that `numpy` version: [setup.py v 0.54](https://github.com/numba/numba/blob/release0.54/setup.py) vs [setup.py master](https://github.com/numba/numba/blob/master/setup.py) vs [setup.py v0.53](https://github.com/numba/numba/blob/release0.53/setup.py)

Reply:
I see. Let's keep it like this, then.